packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,364 @@
+<!--
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>&lt;model-viewer&gt; PostProcessing Examples</title>
```

Comment:
Do these examples still render transparently? As in DOM content behind MV shows through? If not, we may want to mention what limitations users should be aware of. I imagine this might be particularly noticeable for bloom.

Reply:
![image](https://user-images.githubusercontent.com/54665621/229703284-2fcac9e7-d31d-4010-a679-089fdc84dd0e.png)

Transparency still works for all effects